src/main/resources/config.yml

```diff
@@ -14,6 +14,10 @@ sections:
     world: world
 # The damage players will take when outside a managed section.
 damageOutsideSections: 1.0
+# The maximum safe falling distance, after which players will start taking damage. Set to -1 to disable falling damage
+maxSafeFallingDistance: 20.0
```

Comment:
I think these should default to what the defaults are in code, so -1 and 0 would probably be best.

Reply:
Also just in case you're wondering where we actually manage the configs that go on the server, that'll be https://github.com/MineInAbyss/server-config, I should get started on moving stuff into here hehe.